js/viz/funnel.js

```diff
@@ -1,13 +1,14 @@
 "use strict";
 
-var Funnel = require("./funnel/funnel");
+var dxFunnel = require("./funnel/funnel");
 
-Funnel.addPlugin(require("./funnel/label").plugin);
-Funnel.addPlugin(require("./core/export").plugin);
-Funnel.addPlugin(require("./core/title").plugin);
-Funnel.addPlugin(require("./components/legend").plugin);
-Funnel.addPlugin(require("./funnel/tracker").plugin);
-Funnel.addPlugin(require("./funnel/tooltip").plugin);
-Funnel.addPlugin(require("./core/loading_indicator").plugin);
+dxFunnel.addPlugin(require("./funnel/label").plugin);
+dxFunnel.addPlugin(require("./core/export").plugin);
+dxFunnel.addPlugin(require("./core/title").plugin);
+dxFunnel.addPlugin(require("./components/legend").plugin);
+dxFunnel.addPlugin(require("./funnel/tracker").plugin);
+dxFunnel.addPlugin(require("./funnel/tooltip").plugin);
+dxFunnel.addPlugin(require("./core/loading_indicator").plugin);
 
-module.exports = Funnel;
+registerComponent("dxFunnel", dxFunnel);
```

Comment:
Component already is registered (see viz/funnel/funnel,js)

Reply:
Fixed